.gitignore

```diff
@@ -14,4 +14,5 @@ config/configure.in
 x-tools
 .config.*
 *.bz2
-
+lazy-newb-pack-*.src.rpm
```

Comment:
We could call the package "linux-dwarf-pack-*", since it goes by this name on the bay12 forum and on DFFD.